var a = 10;
_assertEqual("1", "1");
function bad($run){
	_log("Run "+ $run);
	if (a == 10){
		try{
			_assertsEqual("123", "45");
			y = x + 1;
		} catch (e) {
			_log("In catch");
			_logExceptionAsFailure(e);
			//throw e;
		}
	}
}
bad(1);
bad(2);
a = 93;
b = 45;